views/includes/check-your-answers/presenter.html

```diff
@@ -1,6 +1,16 @@
 <h2 class="govuk-heading-l">Contact details</h2>
 <h3 class="govuk-heading-m">Who can we contact about this application?</h3>
 
+{% set presenterNameLink = "" %}
+{% set presenterEmailLink = "" %}
+{% if pageParams.isRegistration %}
+  {% set presenterNameLink = OE_CONFIGS.PRESENTER_CHANGE_FULL_NAME %} 
+  {% set presenterEmailLink = OE_CONFIGS.PRESENTER_CHANGE_EMAIL %}
+{% else %} 
+  {% set presenterNameLink = OE_CONFIGS.UPDATE_PRESENTER_CHANGE_FULL_NAME %} 
+  {% set presenterEmailLink = OE_CONFIGS.UPDATE_PRESENTER_CHANGE_EMAIL %}
+{% endif %}
+
```

Comment:
is it possible to pass values for `presenterNameLink` and `presenterEmailLink ` through from each register/update controller so you don't have to do the whole if/else in here?

Reply:
I would rather keep changes minimal to update rather than both

Reply:
Name changed to presenterXYZChangeLink format